dpmjetIII191_workaround.py

```diff
@@ -64,4 +69,8 @@
 out = list(map(str, unchanged))
 out += list(map(str, modded))
 
-sys.stdout.write(";".join(out))
+if platform.system() == "Windows":
+    with open(sys.argv[3], "w") as f:
+        f.write(";".join(out).replace("\\", "/"))
+else:
+    sys.stdout.write(";".join(out))
```

Comment:
No need to do it both ways.

Reply:
Do you mean we need to leave only solution with temporary file? Because stdout way doesn't work in Windows.

Reply:
Only solution with temporary file is left.